workflows/prognostic_run_diags/fv3net/diagnostics/prognostic_run/computed_diagnostics.py

```diff
@@ -217,6 +217,22 @@ def _get_metric(self, metric_type: str, variable: str, run: str) -> pd.Series:
         return _metrics[_metrics.run == run]
 
 
+@dataclass
+class RunMovieUrls:
+    """Represents locations of movies for a collection of run diagnostics."""
+
+    movies: Mapping[str, Sequence[str]]  # mapping from run name to sequence of URLs
+
+    def by_name(self):
```

Comment:
The concept of `name` is overloaded.
```suggestion
    def by_movie_name(self):
```